src/SOS/Strike/sos.cpp

```diff
@@ -833,6 +865,7 @@ namespace sos
     void SyncBlk::Init()
     {
         if (FAILED(mData.Request(g_sos, mIndex)))
+
```

Comment:
nit: why add the blank line?

Reply:
It wasn't intentional, I must have accidentally added it at some point